README.md

```diff
@@ -41,6 +41,26 @@ Or with a regex:
 		with    "vm.swappiness=60"
 	end
 
+
+replace_line_or_add
+------------
+
+Search the file line by line and match each line with the given regex if matched, replace the whole line with the replace parameter.
+If no match is found, add entry like the append_line resource
```

Comment:
What happens when the line being added doesn't match the replacement regex?  Will a line be inserted every run?  Append checks to see if the line is already there. I think that behavior would make more sense.